scripts/release_scripts/deploy.py

```diff
@@ -74,6 +74,11 @@
 APP_NAME_OPPIATESTSERVER = 'oppiatestserver'
 BUCKET_NAME_SUFFIX = '-resources'
 
+APP_NAME_TO_REDISHOST_NAME = {
```

Comment:
Better to generalize this so that, at least from this script's perspective, it's not app-name dependent.

Reply:
Not required now. Updated according to release scripts update.